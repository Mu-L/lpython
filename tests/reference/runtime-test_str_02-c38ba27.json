{
    "basename": "runtime-test_str_02-c38ba27",
    "cmd": "lpython {infile}",
    "infile": "tests/runtime_errors/test_str_02.py",
    "infile_hash": "63853d644565a26ea82a41221d8fe11d9fc20d9132a36a76e6aba5e6",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "runtime-test_str_02-c38ba27.stdout",
    "stdout_hash": "58996894a70cea063a6f8fa0d23184ecd067f8756ac12319ca8bbff8",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 1
}