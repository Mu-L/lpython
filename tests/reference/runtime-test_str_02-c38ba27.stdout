%string_descriptor = type { i8*, i64, i64 }
  %casted_string_ptr_to_desc = alloca %string_descriptor, align 8
%string_descriptor*

String index: -8is out of Bounds
